symfony/plugins/orangehrmCorePlugin/Api/V2/RequestParams.php

```diff
@@ -144,6 +146,27 @@ public function getArrayOrNull(string $type, string $key, ?array $default = null
         return $this->$type->get($key, $default);
     }
 
+    /**
+     * @param string $type
+     * @param string $key
+     * @param array|null $default
+     * @return Base64Attachment|null
+     */
+    public function getAttachmentOrNull(string $type, string $key, ?array $default = null): ?Base64Attachment
```

Comment:
getAttachment() is fine

Reply:
this is not required scenario, then we have to allow null